downstream/modules/builder/ref-scenario-using-authentication-ee.adoc

```diff
@@ -22,14 +22,12 @@ additional_build_files:
 
```

Comment:
`additional_build_files` isn't necessary at all for this one now, and unless you want it to be a complete example (in which case it needs `version` and `images`, `options` is also unnecessary.